views/homepage.erb

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+  </head>
+  <body>
+
+    <div class="container mt-3">
+      <h1>Prompter</h1>
+
+      <p>The prompter tool allows you to simply compare data
+        from Wikidata (found from a SPARQL query) with an
+        external CSV file.</p>
+
+      <p>In order to use the prompter, you just add the
+        <a href="https://www.wikidata.org/wiki/Template:Compare_Wikidata_with_CSV"><tt>{{Compare Wikidata with CSV}}</tt></a>
+        template to a page.</p>
+
+      <p>The source code for the prompter tool can be found in
+        these two repositories:</p>
+
+      <ul>
+        <li><a href="https://github.com/everypolitician/prompter">everypoltiian/prompter</a>: the HTTP interface to the tool</li>
+        <li><a href="https://github.com/everypolitician/compare_with_wikidata">everypolitician/compare_with_wikidata</a>: the underlying library that compares the data</li>
+      </ul>
+
+      <p>This tool is being developed as part of the
+        <a href="https://meta.wikimedia.org/wiki/Grants:Project/mySociety/EveryPolitician">MedaWiki
```

Comment:
Oh and another here, "MedaWiki", though I think in this context it should probably be "Wikimedia", the org, rather than "MediaWiki", the software :)